tables/TaQL/TaQLNodeHandler.cc

```diff
@@ -473,23 +473,26 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
       TaQLNodeResult ires1 = visitNode (node.itsIndices1);
       if (node.itsIndices2.isValid()) {
         TaQLNodeResult ires2 = visitNode (node.itsIndices2);
-        topStack()->addUpdate (new TableParseUpdate(node.itsName,
-                                                    node.itsNameMask,
-                                                    getHR(ires1).getExprSet(),
-                                                    getHR(ires2).getExprSet(),
-                                                    expr,
-                                                    node.itsIndices1.style()));
+        topStack()->addUpdate (std::shared_ptr<TableParseUpdate>
+                               (new TableParseUpdate(node.itsName,
+                                                     node.itsNameMask,
+                                                     getHR(ires1).getExprSet(),
+                                                     getHR(ires2).getExprSet(),
+                                                     expr,
+                                                     node.itsIndices1.style())));
```

Comment:
Optional. The other `addUpdate()` calls can also benefit from `std::make_shared`.
```suggestion
        topStack()->addUpdate (std::make_shared<TableParseUpdate>
                                                    (node.itsName,
                                                     node.itsNameMask,
                                                     getHR(ires1).getExprSet(),
                                                     getHR(ires2).getExprSet(),
                                                     expr,
                                                     node.itsIndices1.style()));
```

Reply:
Good point. Changed them.